### ISSUE TITLE
Bad indent HTML tag <a>

### ISSUE THREAD

**Issue by Undefinaids** [beautify-web/js-beautify#1270]
# Description
Hi !
I think i have found a problem with HTML indent with the "a" tag

# Input
The code looked like this before beautification:
```
<body><div class="p0-row-0 row"><a href="#!" id="button-0" class="button-0 waves-effect waves-light btn">ButtonIm</a></div></body>
```

# Expected Output
The code should have looked like this after beautification:
```
<body>
        <div class="p0-row-0 row">
                <a href="#!" id="button-0" class="button-0 waves-effect waves-light btn">
                       ButtonIm
                </a>
        </div>
</body>
```

# Actual Output
The  code actually looked like this after beautification:
```
<body>
        <div class="p0-row-0 row">
                <a href="#!" id="button-0" class="button-0 waves-effect waves-light btn">
ButtonIm
</a>
        </div>
</body>
```

# Steps to Reproduce
Copy, paste, enjoy

## Environment
OS: Linux

## Settings
Default settings

**Comment by bitwiseman** [comment]
Is this on http://jsbeautifier.org ?   I'm not get that output.

**Comment by Undefinaids** [comment]
Hi, this happened when using the js-beautify module not by using http://jsbeautifier.org/

**Comment by bitwiseman** [comment]
Please see if you can reproduce this using the website. or otherwise include the settings you are using.

**Comment by Undefinaids** [comment]
Ok,
so i finally did it on the website with the default parameters !
The input is : 
```
<div class="col s3">
    <a href="#!" id="button-0" class="style-1 waves-effect waves-light btn">
      Button 1
</a>
</div>
```

And the output is : 
```
<div class="col s3">
    <a href="#!" id="button-0" class="style-1 waves-effect waves-light btn">
      Button 1
</a>
</div>
```

I tried many times with different input and i found that everything become well indented expect  '</a>'
I hope you'll have the same output as me ! 
Thank you for your attention

**Comment by bitwiseman** [comment]
@Undefinaids Yes, that is that same output I got.  I think the stuff inside the `<a>` element is "unformatted", so if you add indents it will stay there.

**Comment by bitwiseman** [comment]
Fixed in 1.7.6.
